packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.styles.ts

```diff
@@ -0,0 +1,22 @@
+import { ILayerStyles, ILayerStyleProps } from '../../Layer';
+import { IKeytipLayerStyleProps, IKeytipLayerStyles } from './KeytipLayer.types';
+import { hiddenContentStyle } from '../../Styling';
+
+export const getLayerStyles = (props: ILayerStyleProps): ILayerStyles => {
+  return {
+    root: [{
+      // Prioritize the Keytips above all other Layers
+      zIndex: 1000001
```

Comment:
Hmmm, is there no better way to do this? Are we sure keytips should always be on top?